extensions/linear/src/api/linearClient.ts

```diff
@@ -10,6 +10,7 @@ export const linear = OAuthService.linear({
       accessToken: token,
       headers: {
         "public-file-urls-expire-in": "60",
+        "linear-raycast-extension": "true",
```

Comment:
Nit (non-blocking): I believe this is to disambiguate requests on linear server-side 😎 . Should we use `extensionName` and `commandName` from [environment](https://developers.raycast.com/api-reference/environment) for more granularity? I think commandName can get finicky in case the client is being initialized only once.